pkg/runner/step_action_remote.go

```diff
@@ -50,6 +50,14 @@ func (sar *stepActionRemote) pre() common.Executor {
 				return nil
 			}
 
+			sar.remoteAction.URL = sar.RunContext.Config.GitHubInstance
+			for _, throughAction := range sar.RunContext.Config.ThroughAction {
+				if (sar.remoteAction.Org + "/" + sar.remoteAction.Repo) == throughAction {
```

Comment:
Please use string.EqualFold instead of ==, repository names are case insensitive